README.md

```diff
@@ -0,0 +1,92 @@
+IRBEM Library
+=============
+
+The International Radiation Belt Environment Modeling (IRBEM) library is
+freely distributed under the umbrella of the PRBEM COSPAR panel.
+
+In 2003 ONERA-DESP (space environment department) decided to put together a
+set of source codes into a library dedicated to radiation belt modeling. The
+toolkit was then called ONERA-DESP-LIB. Because the project has grown along
+time, and because its development is nowadays more like an international
+collaborative effort, it has been decided in 2008, after COSPAR 2008
+Montreal assembly, to move the library name to IRBEM-LIB (which refers to
+COSPAR PRBEM panel) and to distribute it under COSPAR PRBEM umbrella (a
+neutral body).
+
+The IRBEM Fortran library allows to compute magnetic coordinates and drift
+shells using various external magnetic field models.  Further routines are
+provided for  various coordinate and time format transformations.
+
+The library can be called from FORTRAN or C code and from IDL, Python or
+MATLAB code. For IDL, Python and MATLAB wrappers are provided in the
+distribution package. 
+
+Installation
+------------
+IRBEM requires a Fortran compiler, and can be installed on most
+environments.
+
+Quick install procedure on Linux with gfortran:
+
+    git clone https://github.com/PRBEM/IRBEM.git
+	cd IRBEM
+	make OS=linux ENV=gnu64 install
```

Comment:
These instructions are missing the ```make OS=linux64 ENV=gnu64 all``` command. Also the `linux` should be `linux64` in the ```make...install``` command

Reply:
Yes thanks, I have corrected the Readme. It should now compile (after applying this PR and #5 ) out of the box when following the instructions.